commons/dto/NotificationDTO.ts

```diff
@@ -18,3 +18,30 @@ export type NotificationDTO = DTO & HasIdentifier & {
   payload?: Record<string, unknown>;
   createdAt: string;
 }
+
+export type BloodDonationNotificationDTO = DTO & HasIdentifier & {
+  id: string;
+  userId: string;
+  title: string;
+  body: string;
+  type: NotificationType;
+  status: NotificationStatus;
+  payload: {
```

Comment:
Can't we construct payload from existing type?

Reply:
Updated in 49f319a8110867cd9a5be505206104734474bfa3